tests/e2e/script_test.go

```diff
@@ -36,6 +38,14 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+var runScenarios string
```

Comment:
i think i want this to be a repeatable flag;

then we can run a bunch of pause related scenarios under the pause CI step

Reply:
My _personal_ view is that (1) flags are a pain to pass to tests, and we want (2) `go test ./...` to cover the "core" scenarios.  If you want to override a kubeconfig, by all means use an env var or a flag, but `go test ./...` should be what we run in the default CI.

In this case, I think the `-run`  flag on `go test` does this?